mirrors.d/us.mirrors.cicku.me.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: us.mirrors.cicku.me
+address:
+  http: http://us.mirrors.cicku.me/almalinux
+  https: https://us.mirrors.cicku.me/almalinux
+geolocation:
+  country: US
+  state_province: 
+  city: 
```

Comment:
Where is this one?

Reply:
Any colocation available in the contiguous US: https://www.cloudflarestatus.com/

Reply:
We need a specific location per mirror entry.

Reply:
Let me pull the mirror list and see which city can be utilized the best

Reply:
Done